src/worker/github_similar_content.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any
+
+import asyncpg
+import logfire
+from httpx import AsyncClient
+from pydantic import BaseModel, Field
+from pydantic_ai import Agent
+
+from ..common.db.github import GithubContentProject, fetch_issues_for_similarity_check, find_similar_issues
+
+
+class SimilarityResult(BaseModel):
+    percentage: int = Field(description='Similarity of the issues', ge=0, le=100)
```

Comment:
probably useful to have the two issues being compared here also (links and titles)

Reply:
I am not very familiar with `pydantic-ai`, but this is the result model. also, what would be the benefit of the issue links and titles here?

Reply:
Also useful to have a `reason` section (make sure the few shot examples in the prompt match)